CMakeLists.txt

```diff
@@ -1,16 +1,27 @@
 cmake_minimum_required(VERSION 3.5)
 project(LibreCuda CXX)
 
-# create elf-parser library
-add_subdirectory(third_party/ELFIO)
+set( CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin )
+set( CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib )
 
-add_executable(LibreCuda
-        src/main.cpp
-        src/librecuda.cpp
-        src/librecuda_status.cpp
-        src/cmdqueue.cpp)
 
```

Comment:
C++ 17 is required for ELFIO to build.
Without set(CMAKE_CXX_STANDARD 17) here, this doesn't build.

Reply:
https://github.com/mikex86/LibreCuda/pull/11/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR7-R15